types/resource.go

```diff
@@ -198,10 +198,23 @@ const (
 	NodeStateDown = NodeState("down")
 )
 
+type NodeConditionType string
+
+const (
+	NodeConditionTypeMountPropagation = "MountPropagation"
+	NodeConditionTypeNodeState        = "NodeState"
+	NodeConditionTypeDiskState        = "DiskState"
+)
+
+const (
+	NodeConditionReasonNotSupportMountPropagation = "NotSupportMountPropagation"
```

Comment:
`NoMountPropagationSupport`.

Otherwise you need to do `DontSupportMountPropagation` which is longer.